lib/matplotlib/projections/polar.py

```diff
@@ -20,39 +20,50 @@ class PolarTransform(mtransforms.Transform):
     The base polar transform.
 
     This transform maps polar coordinates ``(theta, r)`` into Cartesian
-    coordinates ``(x, y) = (r * cos(theta), r * sin(theta))`` (but does not
+    coordinates ``(x, y) = (r * cos(theta), r * sin(theta))``
```

Comment:
It may be nice to get this in math-mode, although not required.

Reply:
👍 done